packages/telemetry/src/slog-to-otel.js

```diff
@@ -66,11 +77,7 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
   let currentAttrs = {};
 
   const cleanAttrs = attrs => ({
-    ...Object.fromEntries(
-      Object.entries(attrs)
-        .map(([key, value]) => [`agoric.${key}`, cleanValue(value, key)])
-        .filter(([_key, value]) => value !== undefined && value !== null),
-    ),
+    ...serializeInto(attrs, 'agoric'),
```

Comment:
@michaelfig this does cause some data to be serialized deeply, such as `slots` being written as `.slots.0=ko123` etc instead of `.slots=["ko123"]`

Reply:
That's perfectly fine, IMHO.

Reply:
That sounds like it makes the number of attributes per (span?) become larger and variable.. if some crank uses a thousand slots, is that going to cause problems on the telemetry provider?

While it's nice to be able to recover the list of slots, I'm hard pressed to imagine a honeycomb-side search that could use it (slogfile-processing tools are probably more useful there). Unless it's possible to do a honeycomb search like "find me all spans WHERE any `.slots.*` attributes have value `ko123`", which would be kinda cool for causality tracing (currently I do this with a lot of `jq` work and/or ad-hoc python tools).

Reply:
Right, a base `slots` which is a comma separated string of values may be more useful, and could be used with `contains` in Honeycomb queries

Reply:
I explicitly handled `slots` below to avoid the splitting in separate fields.